src/components/views/Page.tsx

```diff
@@ -21,12 +21,12 @@ const PageView = ({
                 {breadcrumbs && (
                     <div className="flex gap-1">
                         {breadcrumbs.map((breadcrumb, index) => (
-                            <>
-                                <Link key={index} href={breadcrumb.href}>
+                            <div key = {index} className="flex flex-row">
```

Comment:
made this change as I was getting an error about not having a key prop for each child